### PR TITLE
copy old fulfillments data from old bounty to new instead of recreate…

### PR DESCRIPTION
**Pull request by danlipert**
##### Description

This PR fixes a bug where fulfillments data is not copied over from an old bounty because the window is closed when work is submitted or a bounty is updated

##### Refers/Fixes

reported in slack

##### Testing

untested

### CODE REVIEW

**Comment by thelostone-mc** [comment]
image

RESPONSE

Screenshot 2020-01-28 at 8 47 57 PM

**Comment by thelostone-mc** [comment]
@danlipert updated it as per review comments and tested it again

screencapture-localhost-8000-issue-gitcoinco-web-5122-2603-2020-01-28-20_44_08

File: app/dashboard/helpers.py [hunk 1]
```diff
@@ -628,15 +659,14 @@ def merge_bounty(latest_old_bounty, new_bounty, metadata, bounty_details, verbos
...
-    # migrate fulfillments, and only take the ones from
-    # fulfillments metadata will be empty when bounty is first created
     fulfillments = bounty_details.get('fulfillments', {})
     if fulfillments:
         handle_bounty_fulfillments(fulfillments, new_bounty, latest_old_bounty)
         url = normalize_url(new_bounty.github_url)
         for inactive in Bounty.objects.filter(
             current_bounty=False, github_url=url
         ).nocache().order_by('-created_on'):
+            # TODO: evalute if this can be removed
```
Comment (danlipert) [inline app/dashboard/helpers.py]: As far as I can tell, this filter should always return nothing, right?

Comment (thelostone-mc) [reply app/dashboard/helpers.py]: Yes that is correct ! It's just a fallback code

Comment (danlipert) [reply app/dashboard/helpers.py]: Meh lets just leave it I guess

**Comment by danlipert** [comment]
https://github.com/gitcoinco/web/pull/5945

**Pull request closed without merging**